### PR TITLE
bug 1613077 updating files to back up

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1613077

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@jianlinliu PTAL

**Comment by kalexand-rh** [comment]
@jiajliu, PTAL

**Comment by vikram-redhat** [comment]
@kalexand-rh - does this need to be in 3.11 as well?

**Comment by kalexand-rh** [comment]
@jiajliu approved this change in the source bug. (I made a small change to the text in the step to improve the clarity - only some of the files are deleted!)

@openshift/team-documentation PTAL

File: upgrading/automated_upgrades.adoc [hunk 1]
```diff
@@ -286,15 +286,19 @@ requirements.
...
 /usr/lib/systemd/system/atomic-openshift-master-api.service
 /usr/lib/systemd/system/atomic-openshift-master-controllers.service
 /etc/sysconfig/atomic-openshift-master-api
 /etc/sysconfig/atomic-openshift-master-controllers
+/etc/origin/master/master-config.yaml
+/etc/origin/master/scheduler.json
 ----
++
+The *systemd* files are removed during the upgrade to the control plane static
+pod architecture.
```
Comment (bfallonf) [inline upgrading/automated_upgrades.adoc]: @kalexand-rh Does this mean:

"The systemd files that are removed during the upgrade are sent to the control plane static pod architecture."

or

"The systemd files are removed when you perform the upgrade to the control plane static pod architecture."

?

**Comment by bfallonf** [comment]
@kalexand-rh One question. Otherwise, LGTM :ramen:

**Pull request merged**